packages/app/src/app/overmind/effects/live/index.ts

```diff
@@ -171,6 +187,10 @@ class Live {
             // If we can't get a jwt because we're unauthorized, disconnect...
             this.socket.disconnect();
             tries = 0;
+          } else if (error.response?.status === 503) {
+            isServerDown = true;
+          } else {
+            isServerDown = false;
```

Comment:
I would rather set `isServerDown = false` somewhere else. Is there a "connected" event or similar? It seems risky to only say server is up again based on errors?

Reply:
Yep I think I should make it part of a separate if statement. I do think this `catch` is the place, because this reconnect happens when the server disconnects (eg. goes down), if the server returns a 503 on reconnect it means that the server is down (since we return that status code if we can't reach the server from our proxy). So this is our way to ensure that we don't flood on the next try.